app/api/proofs/route.ts

```diff
@@ -0,0 +1,87 @@
+import type { NextRequest } from "next/server"
+import { z } from "zod"
+
+import { fetchProofsFiltered } from "@/lib/api/proofs"
+import { getProvingCost } from "@/lib/proofs"
+import { ProofWithCluster } from "@/lib/types"
+
+const queryParamsSchema = z.object({
+  team: z.string().optional(),
+  block: z.coerce.number().int().positive().optional(),
+  limit: z.coerce.number().int().positive().max(1000).default(100),
+  offset: z.coerce.number().int().min(0).default(0),
+})
+
+export async function GET(request: NextRequest) {
```

Comment:
What is this route for? Aren't you just needing to expose external endpoints?

Reply:
Dangling file -- fixed.